sunrisemvmtnyc/legislation-tracker
# feat: Fetch sunrise bills/categories instead of hard-coded options

- [x] add missing bills to airtable

This fetches the bill IDs & campaigns from the airtable, and has a new endpoint to show them. _Only backend_, no frontend implementation atm.

### To test:

Make sure you have `AIRTABLE_API_KEY` in your `.env` file (see directions in #31). Run & visit http://localhost:5173/api/v1/bills/airtable-bills. Verify that:
- each is keyed by a senate or assembly bill ID
- `A01263` has two campaigns

### Reference:

airtable table: https://airtable.com/appBS7e3igXNBXl9b/tblCJq2YX259pH5ij/viwMwbZUFjBlA9KLK
airtable bespoke table API reference: https://airtable.com/appBS7e3igXNBXl9b/api/docs
airtable base API reference: https://airtable.com/developers/web/api/introduction

-----------------

### DEPRECATED - kept for reference only

- [x] remove google sheets logic
- [x] remove google sheets js library
- [x] remove google sheets env var

file:///Users/mymac/Developer/oss/google/google-api-nodejs-client/docs/sheets/classes/Resource$Spreadsheets$Values.html#get
https://developers.google.com/sheets/api/guides/concepts
https://developers.google.com/sheets/api/quickstart/nodejs
https://developers.google.com/sheets/api/guides/values#node.js
